tests/Doctrine/Tests/Common/Annotations/Ticket/DCOM58Test.php

```diff
@@ -16,7 +16,11 @@ class DCOM58Test extends \PHPUnit_Framework_TestCase
     public function testIssue()
     {
         $reader     = new AnnotationReader();
+
+        self::assertTrue(class_exists(\Doctrine\Common\Annotations\Annotation\IgnoreAnnotation::class), false);
```

Comment:
You're passing `false` as message to `assertTrue`. Did you maybe want to pass it to `class_exists`?

Reply:
Also, import the class instead of using the FQCN.